.github/workflows/ContinuousIntegration.yml

```
@@ -2,11 +2,13 @@
 name: Continuous Integration
 
 # This workflow is triggered on pushes and pull requests to the repository.
-on: 
+on:
   push:
     branches: '*'
```

```suggestion
    branches: '**'
```
Two asterisks are needed for branch names that have a `/` in them.